Makefile

```diff
@@ -1,12 +1,27 @@
+#
+# on CentOS 6.5 call with
+#    scl enable devtoolset-7 -- make
+#
 NAME = myprogram
-CFLAGS = -std=c++14 $(pkg-config --cflags --libs libpistache)
+
+CFLAGS += -c -std=c++14 $(shell PKG_CONFIG_PATH=/usr/local/lib64/pkgconfig/ pkg-config --cflags libpistache)
```

Comment:
Note that PKG_CONFIG_PATH is required only if pistache is built and installed locally.
for the PPA install it should not be used.
maybe try to get cflags and libs from default pkg-config, and if they are empty - try to get from /usr/local/ ?